client/components/with-fallback-styles/util.js

```diff
@@ -37,3 +37,81 @@ export const getBorderColor = ( borderNode ) => {
 		? window.getComputedStyle( borderNode ).borderBlockStartColor
 		: null;
 };
+
+/**
+ * External dependencies
+ */
+import fastDeepEqual from 'fast-deep-equal/es6';
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+import { createHigherOrderComponent } from '@wordpress/compose';
+
+export const withWordPressFallbackStyles = ( mapNodeToProps ) =>
```

Comment:
Good call on moving this method here! 👍

Given the way that Gutenberg recommends externs for WP and how big `@wordpress/components` is, it's really not a great idea to have utility methods in there 😕